scripts/test-unit.sh

```diff
@@ -1,12 +1,11 @@
 #!/bin/bash
-#
-# Copyright 2016 The Kubernetes Authors.
```

Comment:
Do we actually own this or did we copy it?
If we copied it, we need to retain the copyright header and add our copyright line underneath.

Reply:
There's not much left of this file other than calling other local scripts, so I don't think there's much to be copied. I haven't dug into the history of this file